target/linux/ramips/dts/mt7621_d-team_pbr-m1.dts

```diff
@@ -60,22 +60,30 @@
 		};
 	};
 
+	reg_usb_vbus: regulator-usb {
+		compatible = "regulator-fixed";
+		regulator-name = "usb_vbus";
+		regulator-min-microvolt = <5000000>;
+		regulator-max-microvolt = <5000000>;
+		gpios = <&gpio 25 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+		regulator-boot-on;
```

Comment:
I do not think this is needed. The documentation says:
```
  regulator-boot-on:
    description: bootloader/firmware enabled regulator.
      It's expected that this regulator was left on by the bootloader.
      If the bootloader didn't leave it on then OS should turn it on
      at boot but shouldn't prevent it from being turned off later.
      This property is intended to only be used for regulators where
      software cannot read the state of the regulator.
    type: boolean
```
https://www.kernel.org/doc/Documentation/devicetree/bindings/regulator/regulator.yaml

The driver should be able to read the state of the GPIO.

Reply:
In the fixed-regulator. example they set it too: 
https://www.kernel.org/doc/Documentation/devicetree/bindings/regulator/fixed-regulator.yaml

Reply:
Removed.